en/docs/guides/federation/facebook.md

```diff
@@ -0,0 +1,120 @@
+# Configure Facebook as a Federated Authenticator
+
+This page guides you through configuring Facebook as a federated authenticator in WSO2 Identity Server. 
+
+---
+
+This guide assumes you have your own applications. If you wish to try out this flow with a sample application, click the button below. 
+
+<a class="samplebtn_a" href="../../../quick-starts/fb-as-federated-authenticator-sample" target="_blank" rel="nofollow noopener">Try it with the sample</a>
+
+---
+
+!!! tip "Before you begin"
+	
+	1.	[Install](https://wso2.com/identity-and-access-management/install/) WSO2 Identity Server.
+	
+	2.	Navigate to `<IS_HOME>/bin` and run the product using one of the following commands. For more information, see [Running the product](insert-link).
+
+		```java tab="Windows"
+		wso2server.bat
+		```
+
+		```java tab="Linux/MacOS"
+		wso2server.sh
+		```
```

Comment:
At one point we discussed that we would not start every page with download and run instructions...if they used quick installer, these run commands would not be applicable as well. 

Shall we remove the tip from here and link to a page with instructions on the first step where the user has to use the dev portal?

Reply:
same comment for other federation guides as well